infra/e2e/turing.values.yaml

```diff
@@ -18,6 +18,7 @@ turing:
     type: rpc-plugin
     rpcPlugin:
       image:  # Value will be set at install time using helm --set command
+      livenessPeriodSeconds: 10
```

Comment:
If introducing the liveness default at the `infra/charts/turing/templates/_helpers.tpl` level, we can just comment this one out here and note that the value can be overridden here.